tests/impl.cpp

```diff
@@ -9393,6 +9395,15 @@ result_t test_mm_set_denormals_zero_mode(const SSE2NEONTestImpl &impl,
     return TEST_SUCCESS;
 }
 
+result_t test_rdtsc(const SSE2NEONTestImpl &impl, uint32_t iter)
+{
+    uint64_t start = _rdtsc();
+    for (int i = 0; i < 100000; i++)
+        ;
```

Comment:
Add an explicit compiler barrier here. That is, `asm volatile("" ::: "memory");`
Reference: https://preshing.com/20120625/memory-ordering-at-compile-time/